src/components/Popups/PopupItem.tsx

```diff
@@ -7,25 +7,28 @@ import { PopupContent } from '../../state/application/reducer'
 import FailedNetworkSwitchPopup from './FailedNetworkSwitchPopup'
 import TransactionPopup from './TransactionPopup'
 
-const StyledClose = styled(X)`
+const StyledClose = styled(X)<{ padding: number }>`
```

Comment:
```suggestion
const StyledClose = styled(X)<{ $padding: number }>`
```

Reply:
ty, ended up changing from padding to a boolean due to zach's comment but will keep in mind for future